package.json

```diff
@@ -48,7 +48,7 @@
     "watch:client:js": "watchify frontend/main.js -o public/js/bundle.js -t [ babelify --presets [ es2015 react stage-2 ] ]",
     "test": "npm run test:prepare && npm run test:server && npm run test:client",
     "test:prepare": "NODE_ENV=test npm run migrate:up && npm run build",
-    "test:server": "NODE_ENV=test mocha --timeout 6000 test/api/bootstrap.test.js test/api/unit/**/*.test.js test/api/requests/**/*.test.js",
+    "test:server": "NODE_ENV=test mocha --timeout 1000 test/api/bootstrap.test.js test/api/unit/**/*.test.js test/api/requests/**/*.test.js",
```

Comment:
That's a sneaky change I made b/c I was frustrated waiting on the 6 second timeout. Now that Sails is in our past most tests run in under 100ms and all should be under 1 second.